src/convertToObject.js

```diff
@@ -6,7 +6,44 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const splitArray = sourceString.trim().split(';');
+  const object = {};
```

Comment:
The variable `object` is not descriptively named, which can decrease code readability. Consider a more meaningful name like `styleObject` or `cssStyles`.